common/frame.go

```diff
@@ -746,15 +746,28 @@ func (f *Frame) Content() string {
 	f.log.Debugf("Frame:Content", "fid:%s furl:%q", f.ID(), f.URL())
 
 	rt := f.vu.Runtime()
-	js := `let content = '';
-		if (document.doctype) {
-			content = new XMLSerializer().serializeToString(document.doctype);
-		}
-		if (document.documentElement) {
-			content += document.documentElement.outerHTML;
-		}
-		return content;`
-	return f.Evaluate(rt.ToValue(js)).(string)
+	js := `() => {
+			let content = '';
+			if (document.doctype) {
+				content = new XMLSerializer().serializeToString(document.doctype);
+			}
+			if (document.documentElement) {
+				content += document.documentElement.outerHTML;
+			}
+			return document.documentElement.outerHTML;
```

Comment:
Should this be `return content;`?

Reply:
Maybe it would be worth creating two tests that traverses both branches of the `if` statement?

Reply:
Whoops, nice catch! I was experimenting with something and this slipped through. :man_facepalming:

Good idea about the test, will add.

Reply:
Fixed in e311e44 and e8d78b4.

I opted for just adding a doctype to the content in the same test, as that hits both branches.